### PR TITLE
Fix cluster creation error due to moving to dataproc python client v2

### PR DESCRIPTION
**Pull request by m-mayran**
- Updated parameters passed to updated classes of the dataproc python client.
- Updated 'JUPYTER' and 'ANACONDA' for Component.JUPYTER and Component.ANACONDA
- Added a local script to quickly test the deployment of JupyterHub locally.

### CODE REVIEW

**Comment by m-mayran** [comment]
Fixes #24

File: dataprocspawner/spawner.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
 from google.cloud.dataproc import ClusterControllerClient
```
Comment (JerryLeiDing) [inline dataprocspawner/spawner.py:25]: Should we change all of these to be google.cloud.dataproc_v1beta2.<class>? I'm wondering if we're kind of mixing between these two.

File: dataprocspawner/spawner.py [hunk 2]
```diff
@@ -25,6 +25,7 @@
...
 from google.cloud.dataproc import ClusterStatus
 from google.cloud.dataproc_v1.services.cluster_controller.transports import ClusterControllerGrpcTransport
+from google.cloud.dataproc_v1.types.shared import Component
```
[the remark this answers is not in the document]
Comment (m-mayran) [reply dataprocspawner/spawner.py]: Updated in commit bac00af

File: dataprocspawner/spawner.py (hunk 1 above)
Comment (m-mayran) [reply dataprocspawner/spawner.py]: Updated in commit bac00af

**Review by JerryLeiDing — APPROVED** [review]

**Pull request merged**